<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mapperDialog</class>
 <widget class="QDialog" name="mapperDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The mapper requires additional configuration. Please make the appropriate changes below.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="pageBox">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>90</height>
      </size>
     </property>
     <property name="title">
      <string>Page Size</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="smallSizeButton">
         <property name="text">
          <string>&amp;800 x 600</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="largeSizeButton">
         <property name="text">
          <string>1&amp;280 x 960</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="mediumSizeButton">
         <property name="text">
          <string>&amp;1024 x 768</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="customSizeButton">
         <property name="text">
          <string>&amp;Custom</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <layout class="QHBoxLayout" name="customSizeLayout">
         <item>
          <widget class="QLabel" name="widthLabel">
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="widthValue">
           <property name="text">
            <string>800</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="heightLabel">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="heightValue">
           <property name="text">
            <string>600</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="detailsLayout">
     <item>
      <widget class="QGroupBox" name="mapLayoutBox">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>110</height>
        </size>
       </property>
       <property name="title">
        <string>Map Layout</string>
       </property>
       <widget class="QRadioButton" name="mollweideButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>102</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&amp;Mollweide</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="gnomicButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&amp;Gnomic</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="atoffButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>85</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Atof&amp;f</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="scaleBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>110</height>
        </size>
       </property>
       <property name="title">
        <string>Color Scale</string>
       </property>
       <widget class="QRadioButton" name="linearButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Linear</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="logButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Log</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="sLogButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Super Log</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="longBox">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Longitude Convention</string>
       </property>
       <widget class="QRadioButton" name="astroButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&amp;Astronomical</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="terrestrialButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>102</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&amp;Terrestrial</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>30</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>mapperDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mapperDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
